pyglider/plotting.py

```diff
@@ -223,55 +363,78 @@ def grid_plots(fname, plottingyaml):
         tmean = ds.temperature.mean(axis=1)
         indmax = np.where(~np.isnan(tmean))[0][-1]
         depmax = ds.depth[indmax]
-        fig, axs = plt.subplots(math.ceil(N/2), 2, figsize=(7.5, 7),
+       # fig, axs = plt.subplots(math.ceil(N/2), 2, figsize=(7.5, 7),  #jpnote changed
+        #                        sharex=True, sharey=True)
+        fig, axs = plt.subplots(int(N / 2), 2, figsize=(7.5, 7),
                                 sharex=True, sharey=True)
         axs = axs.flat
         for n, k in enumerate(keys):
             _log.debug(f'key {k}')
 
             pconf = config['pcolor']['vars'][k]
             _log.debug(pconf)
-            cmap = pconf.get('cmap', 'viridis')
+
+            cmap = plt.cm.get_cmap('viridis') #jp
             vmin = pconf.get('vmin', None)
             vmax = pconf.get('vmax', None)
 
             ax = axs[n]
-            locator = mdates.AutoDateLocator(minticks=3, maxticks=7)
-            formatter = mdates.ConciseDateFormatter(locator)
-            ax.xaxis.set_major_locator(locator)
-            ax.xaxis.set_major_formatter(formatter)
+            ax.yaxis.set_tick_params(labelbottom=True) 
+            ax.xaxis.set_tick_params(labelbottom=True)
+           # locator = mdates.AutoDateLocator() 
+           # formatter = mdates.ConciseDateFormatter(locator)
+           # formatter.offset_formats = ['',
+           #                             '',
+           #                             '',]
+           # ax.xaxis.set_major_locator(locator)
+           # ax.xaxis.set_major_formatter(formatter)
 
             min, max = _autoclim(ds[k])
             if vmin is not None:
                 min = vmin
             if vmax is not None:
                 max = vmax
             # make time windows:
+            if ax == axs[0]:
+                vmax = 300
+                cmap.set_over("black") 
+                ax.set_ylim([300, 0])
+
+            vmax = 300
+            cmap.set_over("black") 
```

Comment:
I'm not a huge fan ov "over" being coloured black...

Reply:
It was to differentiate it from the gray bathymetry. What colour would be better?

Reply:
Actually I've just removed it because it was failing the tests.

Reply:
OK - usually the over cover is just the max color of the colormap?  Maybe you are referring to the NaN behaviour, which will just put an empty square.  That is indeed hard to differentiate from bathymetry depth unless you have a bathy estimate from elsewhere.